modules/lightning_features/lightning_workflow/config/install/views.view.moderation_history.yml

```diff
@@ -339,7 +348,16 @@ display:
     position: 1
     display_options:
       display_extenders: {  }
-      path: node/%/moderation-history
+      path: node/%node/moderation-history
+      menu:
```

Comment:
I don't think we had a link to this before. I agree it makes sense, I just wonder if it'll get unwanted attention is we add this now.